fhir2dataset/query.py

```diff
@@ -187,6 +188,18 @@ def _select_columns(self):
                 final_columns.append(f"{resource_alias}:{element.col_name}")
         self.main_dataframe = self.main_dataframe[final_columns]
 
+    @timing
+    def _remove_lists(self):
+        """Remove lists from columns with only single elements
+        """
+        df = self.main_dataframe
+        for column in df.columns:
+            lengths = df[column].str.len()
+            if all(len < 2 for len in lengths):
+                df[column] = df[column].apply(lambda x: x[0])
+                logger.info(f"the column treated is {column}")
```

Comment:
A small suggestion, here we could say 

```
logger.info(f"extraction of the unique element from the lists composing the {column} column")
```

Reply:
Yes it is better, thank you for the review!